### PR TITLE
rebuild with abseil-cpp 20230802.0

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,12 @@ source:
   #   folder: third_party/re2
 
 build:
-  number: 1
+
+  number: 2
+  # Anaconda's s390x does not contain depdendencies for abseil-cpp or re2, this package is skipped for s390x at this time.
+  # Latest protobuf recipe skips win32, so this package does as well.
+  skip: true  # [win32]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: We do not need win32 anymore

**Comment by skupr-anaconda** [comment]
An error on `osx64`:
`$PREFIX/include/absl/synchronization/internal/kernel_timeout.h:94:45: error: unknown type name 'clockid_t'; did you mean 'clock_t'?`

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: removed

**Comment by M-Waszkiewicz-Anaconda** [comment]
> An error on `osx64`: `$PREFIX/include/absl/synchronization/internal/kernel_timeout.h:94:45: error: unknown type name 'clockid_t'; did you mean 'clock_t'?`

skipped OSX, it is just POC if works on linux-64 I will deal with it

**Pull request closed without merging**